ConnectorRevit/ConnectorRevit2020/ConnectorRevit2020.csproj

```diff
@@ -28,6 +28,7 @@
     <PackageReference Include="ModPlus.Revit.API.2020" Version="1.0.0" ExcludeAssets="runtime" IncludeAssets="compile;build" PrivateAssets="all" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\ConnectorCore\BatchUploader.OperationDriver\BatchUploader.OperationDriver.csproj" />
```

Comment:
I'm aware of why this is being referenced and used within the connector, and although not a real blocker, this is something I'm afraid we're going to regret later.

Reply:
In what sense are we likely to regret this?
The connector is the entry point for our plugins, if we have a connector specific dependency, this is the place for it no?